### PR TITLE
Edits to dbt integration page

### PR DESCRIPTION
**Pull request by discdiver**
Fixes a few links and formatting.
Adds a note and links for a few Prefect concepts to help direct new users if resources are needed.

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
  - If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] If this pull request adds new functionality, it includes unit tests that cover the changes
- [ ] If this pull request removes docs files, it includes redirect settings in `mint.json`.
- [ ] If this pull request adds functions or classes, it includes helpful docstrings.

### CODE REVIEW

**Comment by discdiver** [comment]
cc: @taylor-curran

File: docs/integrations/prefect-dbt/index.mdx [hunk 1]
```diff
@@ -5,6 +5,9 @@ title: prefect-dbt
 With `prefect-dbt`, you can trigger and observe dbt Cloud jobs, execute dbt Core CLI commands, and incorporate other tools, such as [Snowflake](integrations/prefect-snowflake/index), into your dbt runs.
 Prefect provides a global view of the state of your workflows and allows you to take action based on state changes.
 
+Prefect integrations may provide pre-built [blocks](/latest/develop/blocks/), [flows](/latest/develop/flows/), or [tasks](/latest/develop/tasks/) for interacting with external systems.
+Block types in this library allow you to do such things as a dbt Cloud job or execute a dbt Core command.
```
Comment (discdiver) [inline docs/integrations/prefect-dbt/index.mdx:9]:
```suggestion
Block types in this library allow you to do such things as run a dbt Cloud job or execute a dbt Core command.
```

Comment (discdiver) [inline docs/integrations/prefect-dbt/index.mdx:9]:
```suggestion
Block types in this library allow you to do such things as run a dbt Cloud job or execute a dbt Core command.
```

File: docs/integrations/prefect-dbt/index.mdx [hunk 2]
```diff
@@ -5,6 +5,9 @@ title: prefect-dbt
 With `prefect-dbt`, you can trigger and observe dbt Cloud jobs, execute dbt Core CLI commands, and incorporate other tools, such as [Snowflake](integrations/prefect-snowflake/index), into your dbt runs.
 Prefect provides a global view of the state of your workflows and allows you to take action based on state changes.
 
+Prefect integrations may provide pre-built [blocks](/latest/develop/blocks/), [flows](/latest/develop/flows/), or [tasks](/latest/develop/tasks/) for interacting with external systems.
+Block types in this library allow you to do such things as run a dbt Cloud job or execute a dbt Core command.
```
Comment (zzstoatzz) [inline docs/integrations/prefect-dbt/index.mdx:9]:
```suggestion
Block types in this library allow you to do things such as run a dbt Cloud job or execute a dbt Core command.
```

**Review by zzstoatzz — APPROVED** [review]
one word swap but lgtm!

File: docs/integrations/prefect-dbt/index.mdx (hunk 2 above)
Comment (discdiver) [reply docs/integrations/prefect-dbt/index.mdx:9]: Looks like either option works: https://ell.stackexchange.com/questions/14544/difference-between-such-things-as-and-things-such-as

**Pull request merged**